src/iccpd/include/mlacp_tlv.h

```diff
@@ -0,0 +1,455 @@
+    uint16_t member_ports_priority;
+    uint8_t flags;
+    /*
+     * RFC 7275
+     * 7.2.4.  mLACP Aggregator Config TLV
+     * [Page 54]
+     */
+    uint8_t agg_name_len;
+    char agg_name[MAX_L_PORT_NAME];
+} __attribute__ ((packed));
+
+typedef struct mLACPAggConfigTLV mLACPAggConfigTLV;
+
+/*
+ * RFC 7275
+ * 7.2.4.  mLACP Port Config TLV
+ * [Page 54]
+ */
+struct mLACPPortConfigTLV
+{
+    ICCParameter icc_parameter;
+    /* [IEEE-802.1AX], Section 5.3.4. */
+    uint16_t port_num;
+    /*
+     * RFC 7275
+     * 7.2.5.  mLACP Port Config TLV
+     * [Page 55]
+     */
+    uint8_t mac_addr[ETHER_ADDR_LEN];
+    /* [IEEE-802.1AX], Section 5.3.5. */
+    uint16_t actor_key;
+    /* [IEEE-802.1AX], Section 5.3.4. */
+    uint16_t port_priority;
+    /* IF-MIB [RFC2863] */
+    uint32_t port_speed;
+    /*
+     * RFC 7275
+     * 7.2.4.  mLACP Port Config TLV
+     * [Page 55]
+     */
+    uint8_t flags;
+    /*
+     * RFC 7275
+     * 7.2.4.  mLACP Port Config TLV
+     * [Page 56]
+     */
+    uint8_t port_name_len;
+    /* IF-MIB [RFC2863] */
+    char port_name[MAX_L_PORT_NAME];
+
+    /* NOS */
+    uint8_t orphan;
+    uint8_t l3_mode;
+} __attribute__ ((packed));
+
+typedef struct mLACPPortConfigTLV mLACPPortConfigTLV;
+
+/*
+ * RFC 7275
+ * 7.2.6.  mLACP Port Priority TLV
+ * [Page 56]
+ */
+struct mLACPPortPriorityTLV
+{
+    ICCParameter icc_parameter;
+    /*
+     * RFC 7275
+     * 7.2.6.  mLACP Port Priority TLV
+     * [Page 57]
+     */
+    uint16_t op_code;
+    /* [IEEE-802.1AX], Section 5.3.4. */
+    uint16_t port_num;
+    /* [IEEE-802.1AX], Section 5.4.6. */
+    uint16_t agg_id;
+    /* [IEEE-802.1AX], Section 5.3.4. */
+    uint16_t last_port_priority;
+    uint16_t current_port_priority;
+} __attribute__ ((packed));
+
+typedef struct mLACPPortPriorityTLV mLACPPortPriorityTLV;
+
+/*
+ * RFC 7275
+ * 7.2.7.  mLACP Port State TLV
+ * [Page 58]
+ */
+struct mLACPPortStateTLV
+{
+    ICCParameter icc_parameter;
+    /* [IEEE-802.1AX], Section 5.4.2.2, item r. */
+    uint8_t partner_sys_id[ETHER_ADDR_LEN];
+    /* [IEEE-802.1AX], Section 5.4.2.2, item q. */
+    uint16_t partner_sys_priority;
+    /* [IEEE-802.1AX], Section 5.4.2.2, item u. */
+    uint16_t partner_port_num;
+    /* [IEEE-802.1AX], Section 5.4.2.2, item t. */
+    uint16_t partner_port_priority;
+    /* [IEEE-802.1AX], Section 5.4.2.2, item s. */
+    uint16_t partner_key;
+    /* [IEEE-802.1AX], Section 5.4.2.2, item v. */
+    uint8_t partner_state;
+    /* [IEEE-802.1AX], Section 5.4.2.2, item m. */
+    uint8_t actor_state;
+    /* [IEEE-802.1AX], Section 5.3.4. */
+    uint16_t actor_port_num;
+    /* [IEEE-802.1AX], Section 5.3.5. */
+    uint16_t actor_key;
+    /* [IEEE-802.1AX], Section 5.4.8 */
+    uint8_t selected;
+    /*
+     * RFC 7275
+     * 7.2.7.  mLACP Port State TLV
+     * [Page 60]
+     */
+    uint8_t port_state;
+    /* [IEEE-802.1AX], Section 5.4.6. */
+    uint16_t agg_id;
+
+    /* NOS */
+    uint8_t orphan;
```

Comment:
portConfigTlv, LacpPortStateTlv, portChannelInfoTlv has field for orphan.  i believe orphan port related sync is not required. also i dont see any code using it. could you please help me understand use of this

Reply:
OK, we will delete these unused codes.